src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_stacks/tests/test_pipeline_creation.py

```diff
@@ -140,3 +140,107 @@ def test_pipeline_creation_outputs_as_expected_when_source_is_codecommit_and_bui
     assert build_stage_action['ActionTypeId']['Provider'] == "CodeBuild"
 
     assert len(build_stage['Actions']) == 1
+
+def test_pipeline_creation_outputs_with_codeartifact_trigger():
```

Comment:
These functions also need two empty lines before their definition. Could you fix that for all of these in this file?